etc/mpcount.service

```diff
@@ -0,0 +1,6 @@
+[Unit]
```

Comment:
as before, where should I put the file? just under multipath dir instead of etc?